### ISSUE TITLE
Check for irreversibality once the transaction has been sent through api.transact?

### ISSUE THREAD

**Issue by sssubik** [EOSIO/eosjs#851]
How do I check if the transaction has reached its irreversible status?

What does this do?
`useLastIrreversible : True`
Does this mean that the api.transact will return only after the irreversibality has been confirmed??
I could not find it in doc, so hopefully I can get some help.

**Comment by LevanIlashvili** [comment]
No, as stated in [v21.0.2 release notes](https://github.com/EOSIO/eosjs/releases/tag/v21.0.2): `adding the useLastIrreversible argument to transaciton methods will use the last irreversible block number for the transaction TAPOS headers.
`
